### PR TITLE
Document nodemon in shop-backend

### PR DESCRIPTION
**Pull request by chuck-dbos**

### CODE REVIEW

File: e-commerce/README.md [hunk 1]
```diff
@@ -384,3 +384,45 @@ Availability of simple testing command invocation is due to the following lines
...
+  },
+```
+
+### Provide `nodemon` Configuration
+There are [numerous approaches for configuring `nodemon`](https://github.com/remy/nodemon?tab=readme-ov-file#config-files).
+This demo uses `nodemon.json` for configuration, as this is a common default.
+
+```json
+{
+  "watch": ["src/","migrations/"],
```
Comment (qianl15) [inline e-commerce/README.md:418]: If we change `migrations/` we also need to run `npx dbos migrate` again.

Comment (chuck-dbos) [reply e-commerce/README.md:418]: Hum.  Two issues there.  One, the doc, two the underlying example.  You think adding `npx dbos migrate` to the command will do it?  I will try it now.

**Review by qianl15 — APPROVED** [review]
Looks good! We can also consider to add a `npx dbos dev` command to Transact.

**Comment by chuck-dbos** [comment]
> Looks good! We can also consider to add a `npx dbos dev` command to Transact.

I think I will add it to the template.  Because you need the .json file to really do anything useful.  Unclear that `npx dbos dev` will be any simpler to work with.

**Pull request merged**